docker-entrypoint-initdb.sql

```diff
@@ -0,0 +1,907 @@
+--
```

Comment:
Instead of this file, is it possible to run `saltcorn reset-schema -f` ? Otherwise there is a duplication of the definition of the database schema, which is also defined in https://github.com/saltcorn/saltcorn/blob/master/packages/saltcorn-data/db/reset_schema.js.